src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -34,8 +36,15 @@ public FranchiseeAndStaffDto signUpFranchiseeAndStaff(@RequestBody @Valid Franch
 
     @PostMapping("/{franchiseeId}/service_areas")
     @ResponseStatus(HttpStatus.CREATED)
-    public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto suburbListPostDto, @PathVariable(value = "franchiseeId") Long franchiseeId) {
+    public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto suburbListPostDto,
+                                         @PathVariable(value = "franchiseeId") Long franchiseeId) {
         return franchiseeService.addDutyAreas(suburbListPostDto, franchiseeId);
     }
 
+    @PostMapping("/{franchiseeId}/accept_orders")
```

Comment:
better to use Get method

Reply:
GET may be more and is used for reading operations (such as search, read, and query operations) between the client and the server. The POST method may be used more when interacting